src/Auth/Startup.cs

```diff
@@ -91,6 +91,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseOpenIddict();
 
+            app.UseOpenIdConnectServer(configuration =>
```

Comment:
You shouldn't add the OIDC server middleware in an app using OpenIddict, as it's already registered for you by `app.UseOpenIddict()`. At best, this second instance won't work (because you don't set `OpenIdConnectServerOptions.Provider`) and at worst, it will collide with OpenIddict if you use the same authorization/token endpoint path.

What are you trying to do exactly?